blocks.js

```diff
@@ -468,12 +468,9 @@ function attachTypeSwitcherActions() {
 	} );
 
 	Object.keys( typeToTag ).forEach( function( type ) {
-		var selector = '.switch-block__block .type-icon-' + type;
-		var button = queryFirst( selector );
-		var label = queryFirst( selector + ' + label' );
-
+		var iconSelector = '.switch-block__block .type-icon-' + type;
+		var button = queryFirst( iconSelector ).parentNode;
```

Comment:
You're right, we could optimize using a single `switch-block__block` event handler. Let's keep it like this while prototyping :)